byway/pom.xml

```diff
@@ -49,19 +49,6 @@
       <version>2.8.6</version>
     </dependency>
 
-    <dependency>
-      <groupId>com.google.appengine</groupId>
-      <artifactId>appengine-api-1.0-sdk</artifactId>
-      <version>1.9.59</version>
-    </dependency>
-    
-    <!-- https://mvnrepository.com/artifact/com.google.flogger/flogger -->
-    <dependency>
-      <groupId>com.google.flogger</groupId>
-      <artifactId>flogger</artifactId>
-      <version>0.5.1</version>
-    </dependency>
```

Comment:
Why did you remove this?

Reply:
It was one of the duplicate dependencies.

Reply:
These are not duplicates, both are needed. Note the `artifactId` is different:

```xml
    <!-- https://mvnrepository.com/artifact/com.google.flogger/flogger -->
    <dependency>
      <groupId>com.google.flogger</groupId>
      <artifactId>flogger</artifactId>
      <version>0.5.1</version>
    </dependency>

    <!-- https://mvnrepository.com/artifact/com.google.flogger/flogger -->
    <dependency>
      <groupId>com.google.flogger</groupId>
      <artifactId>flogger-system-backend</artifactId>
      <version>0.5.1</version>
    </dependency>
```

Reply:
Oh thanks for pointing that out, I didn't look close enough. Added back.